packages/types/src/tax/common.ts

```diff
@@ -73,3 +73,29 @@ export interface FilterableTaxRegionProps
   updated_at?: OperatorMap<string>
   created_by?: string | string[] | OperatorMap<string>
 }
+
+export interface TaxRateRuleDTO {
+  reference_type: string
```

Comment:
I saw in Carlos' PR that this is called reference, might be better to stick to one for consistency.